File: scrimage-core/src/test/scala/com/sksamuel/scrimage/Issue63Test.scala
package com.sksamuel.scrimage

import org.scalatest.{ FunSuite, Matchers }

/** @author Stephen Samuel */
class Issue63Test extends FunSuite with Matchers {

  test("image for issue 63 should parse") {
    Image(getClass.getResourceAsStream("/issue63.png")).width shouldBe 1600
  }
}

File: scrimage-core/src/test/kotlin/com/sksamuel/scrimage/core/Issue63Test.kt
package com.sksamuel.scrimage.core

import com.sksamuel.scrimage.ImmutableImage
import io.kotest.core.spec.style.FunSpec
import io.kotest.matchers.shouldBe

class Issue63Test : FunSpec({

   test("image for issue 63 should parse") {
      ImmutableImage.fromStream(javaClass.getResourceAsStream("/issue63.png")).width shouldBe 1600
   }

})